README.md

```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
-| Extra theme options          |           | √         | √           |
-|                              |           |           |             |
-| **Performance**              |           |           |             |
-| Pagination                   |           | √         | √           |
-| Search (enhanced)            |           | √         | √           |
-| Search (server-side)         |           |           | √           |
-|                              |           |           |             |
-| **Multiple APIs**            |           |           |             |
-| Multiple versions            |           | √         | √           |
-| Multiple APIs                |           |           | √           |
-| API catalog                  |           |           | √           |
-|                              |           |           |             |
-| **Additional features**      |           |           |             |
-| Try-it console               |           | √         | √           |
-| Automated code samples       |           | √         | √           |
-| Deep links                   |           | √         | √           |
-| More SEO control             |           |           | √           |
-| Contextual docs              |           |           | √           |
-| Landing pages                |           |           | √           |
-| React hooks for more control |           |           | √           |
-| Personalization              |           |           | √           |
-| Analytics integrations       |           |           | √           |
-| Feedback                     |           |           | Coming Soon |
-
-Refer to the Redocly's documentation for more information on these products:
-
-- [Portals](https://redocly.com/docs/developer-portal/introduction/)
-- [Reference](https://redocly.com/docs/api-reference-docs/getting-started/)
-- [Redoc](https://redocly.com/docs/redoc/quickstart/intro/)
-
-## Features
 - Responsive three-panel design with menu/scrolling synchronization
-- [Multiple deployment options](https://redocly.com/docs/redoc/quickstart/intro/)
-- [Server-side rendering (SSR) ready](https://redocly.com/docs/redoc/quickstart/cli/#redoc-cli-commands)
+- Support for OpenAPI 3.1, OpenAPI 3.0 and Swagger 2.0
+- [Multiple deployment options](https://redocly.com/docs/redoc/)
+- High-level grouping in side menu with the [`x-tagGroups`](https://redocly.com/docs/api-reference-docs/specification-extensions/x-tag-groups/) specification extension
 - Ability to integrate your API introduction into the side menu
 - [Simple integration with `create-react-app`](https://redocly.com/docs/redoc/quickstart/react/)
-
-  [Example repo](https://github.com/APIs-guru/create-react-app-redoc)
-- [Command-line interface to bundle your docs into a **zero-dependency** HTML file](https://redocly.com/docs/cli/commands/build-docs/)
-- Neat **interactive** documentation for nested objects <br>
```

Comment:
I don't like the wording here but I would keep the gif and the feature.

Reply:
gifs in READMEs are so wrong and we have so many of them. I'll put the text back.

Reply:
what's wrong with gifs in README? I see them very often (sometimes videos) and find them useful but I must be missing something 🤔

Reply:
They're common, but they're inacessible, distracting or seizure-inducing, and if you can't explain what a feature is with words, do you even know what feature you've built? Especially for a developer tool, if the README isn't designed to be read, what else did you not understand about tools and developers??

(I know this is controversial, but really. I don't think it gives the right image at all when it's in the README - but we should totally link to great video content on these features)